app/views/start/choose-an-npq-and-provider.html

```diff
@@ -0,0 +1,9 @@
+{% extends "layout.html" %}
+{% set title = 'Choose an NPQ and provider' %}
+{% block pageTitle %}{{ title }}{% endblock %}
+
+{% block content %}
+
+{% include 'start/_npq-content.html' %}
```

Comment:
Is the name 'npq-content' descriptive enough?

Reply:
thanks @peteryates, I've added another commit to change this - does that look ok?